### ISSUE TITLE
Why is subrosa.io down?

### ISSUE THREAD

**Issue by k0nsl** [subrosa-io/subrosa-client#50]
Any idea as to why or what the reason is for subrosa.io being completely dead?

**Comment by k0nsl** [comment]
I found the answer.

![subrosa-pending-delete01_k0nsl](https://cloud.githubusercontent.com/assets/2517921/7129264/28172772-e25e-11e4-9dbb-668113ff094e.jpg)

(roll eyes)

**Comment by k0nsl** [comment]
Could any of the people privy to the project perhaps give us some insight as to when the network will be back, or if it ever will be back?

**Comment by mva1985** [comment]
this is unfortunate...  tried signing in and it was a no go

**Comment by k0nsl** [comment]
@mva1985
Yes, a strange "screw-up" to not pay for the domain in time. Glad I got a self-hosted server up & running. We used their network as a sort of "beta test" and we were planning to go the self-hosted route anyway. So I suppose this demise sped the process up.

**Comment by mva1985** [comment]
as for privacy I realize self hosted would be the way to go but the people I communicate with aren't interested in that much "work" they need something easy and this allows video chat and sounded really promising.

**Comment by udlive** [comment]
Hello @k0nsl I need help with setup of Self-hosted {Server} 
hope you can spare me a walk through and suggestions on hosting.
Thanks

**Comment by k0nsl** [comment]
Hi there @udlive. Yes, I can help you out if you get stuck at some point. As for hosting, it all depends on what size you expect your server will be, or how you predict it will evolve. In the beginning a KVM with 1GB RAM and preferably a good SSD array for I/O should suffice. You can always upgrade at a later point.
Companies I recommend are [RamNode](http://goo.gl/bCyxcq), [BuyVM](http://goo.gl/okHkPn) and [Linode](http://goo.gl/fpiabg). Those are all very solid providers. If you rather want a dedicated server, let me know and I'll compile a list of providers that have shown themselves to be decent, reliable and solid when it comes to providing a services that has a good network and uptime. Et cetera.

**Comment by dantman** [comment]
The biggest hurdle I see for myself in self-hosting is the SSL certificate.

Setting up a server, getting the subrosa client and server working, and then going through the process and payments to get https working on it. Just to be able to have private video chats.

**Comment by dantman** [comment]
On the installation part, I maybe someone coming up with a docker container for subrosa.

**Comment by participante0** [comment]
In the mean time... https://jumpch.at/
WebRTC uses SRTP as the encryption protocol (like subrosa). You can chat and share files, and there is no need to create an account in order to start a videocall.

**Comment by k0nsl** [comment]
@participante0: do they have a GH? I can't seem to find it.

**Comment by jumpchat** [comment]
Not yet.  Trying to shoot for open sourcing around june/july timeframe.

**Comment by RoPe93** [comment]
Can I run over tor [Tor hidden service] or I2P?
Please answer if you know.

**Comment by thebgland** [comment]
It's back up everyone!

**Comment by k0nsl** [comment]
Good. Either way, I'll continue to use my self-hosted subrosa.

**Comment by animalillo** [comment]
I wonder if someone cares about this project anymore, I put my own network up and running, and submitted a pull request fixing a server bug, easy fix, but noone is merging or comitting anything to the project it seems.

As for the people looking to setup ssl, there is a place where you can get free ssl certificates, and it works with almost all browsers and clients (http://www.startssl.com/).

**Comment by macbroadcast** [comment]
Just came across this thread by googling "jumpchat github" , was´nt even aware subrosa exist, i was using jumchat for quite some time and found it beside room.co one of the best looking and fanciest webrtc implementations, but subrosa looks great too. I tried to install https://github.com/loowid/loowid but struggled with the fact that it was developed on openshift and could not get the startssl cert working, so vmux was the solution i used on my server till now, because i can use the free cloudflare cert.Looking forward to see jumchat get opensourced. Peace great work guys

**Comment by NipponBill** [comment]
Has this project been abandoned? subrosa.io has been offline for days.
http://downforeveryoneorjustme.com/subrosa.io

**Comment by miclill** [comment]
I loved this project! :-(
So sad to hear about the passing.

**Comment by macbroadcast** [comment]
Guys its an opensource project, this means everyone could get the source and set up a server , did you expect from the project founder to run a server forever for free ? :laughing:

**Comment by jumpchat** [comment]
@macbroadcast sorry it's late. dealing w/ a death in the family. i'll try to get the stuff open sourced before the end of the month.

**Comment by macbroadcast** [comment]
Sorry to hear @jumpchat  ,my warmest condolences :pray:  , will appreciate your contribution.  peace

**Comment by miclill** [comment]
@jumpchat sorry for your loss.
@macbroadcast No, that's not what i meant although also unfortunate. I was talking about the end of development.

**Comment by macbroadcast** [comment]
Any news on releasing @jumpchat as opensource ? :wink:

**Comment by animalillo** [comment]
@jumpchat sorry for your loss.

**Comment by k0nsl** [comment]
Sorry to hear @jumpchat !
It's a terrible thing to go through, I know. Best wishes to you!

**Comment by localstyle** [comment]
Hello!

I was thinking that maybe you guys are used to these things enough to have the answer to my question:
I had a couple of accounts on Subrosa.io and I'd just like to know what happened to the personal data (email addresses and passwords associated) linked to them when the project was discontinued. Is all user info likely to have been deleted from the Subrosa servers?
Besides, I know that Subrosa encrypted personal info and that no human could retrieve e.g. a password if a user lost it, but I'm not sure if/how that relates to my question.

By the way, I found another interesting project called Jitsi: https://jitsi.org/ - https://github.com/jitsi

Thanks,
P

**Comment by macbroadcast** [comment]
Hey @jumpchat what happen to your plans to release jumpchat at the end of october 2015 ?

**Comment by k0nsl** [comment]
@macbroadcast: I don't think we can expect much from @jumpchat. The blog hasn't been updated since 2015.
A pity, it looked like it could have been a viable alternative. As it is, I've decided to use [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat/) for the time being :)

**Comment by macbroadcast** [comment]
yep @k0nsl , it seems he dissappeard from the web :flushed: , a whois on the domain brought me to his itunes site https://itunes.apple.com/us/developer/yulius-tjahjadi/id441451235 its really a pity because he implemented an android/IOS/Javascript SDK allready, hope everything is fine with him.

Does Rocket.chat support WebRTC also ?

**Comment by macbroadcast** [comment]
2018 still no release @jumpchat .What happend to you by the way ?
